src/webots/nodes/WbMesh.cpp

```diff
@@ -97,6 +98,9 @@ void WbMesh::createResizeManipulator() {
 }
 
 bool WbMesh::checkIfNameExists(const aiScene *scene, const QString &name) const {
+  if (!mIsCollada)
+    return false;
+
```

Comment:
Useless code, here is why:

`checkIfNameExists()` is called only if `mIsCollada` is `true` on line 161, so why check if it is false?

```suggestion
```

Reply:
Good point, thank you!